src/test/kotlin/com/cosmotech/api/containerregistry/ContainerRegistryServiceTest.kt

```diff
@@ -3,38 +3,34 @@
 package com.cosmotech.api.containerregistry
 
 import com.cosmotech.api.config.CsmPlatformProperties
-import com.github.dockerjava.api.DockerClient
-import com.github.dockerjava.api.exception.NotFoundException
 import io.mockk.every
 import io.mockk.mockk
 import io.mockk.spyk
 import kotlin.test.BeforeTest
 import kotlin.test.Test
 import kotlin.test.assertEquals
+import org.apache.http.client.ClientProtocolException
 import org.junit.jupiter.api.assertThrows
 
-class DockerContainerRegistryClientTest {
+class ContainerRegistryServiceTest {
```

Comment:
Could you please add tests on errors ?
- when `getRepositoryTagList` returns a json string with no tags property
- when `getRepositoryTagList` returns a non json string

+ Add test to cover "everything went well" case :)

Reply:
The latest tag is applied by default to reference an image when you run Docker commands without specifying the tag value
By construction it's a non empty json array if there's an image